### ISSUE TITLE
feature request: allow mapping the same action to multiple keys

### ISSUE THREAD

**Issue by herrsimon** [rvaiya/warpd#104]
While playing around with warpd, the following issue came up: Currently I use the right hand for moving the pointer and the left hand for clicking buttons. To support the bad habit of holding a beverage or a fork in one hand while navigating a webpage or watching a video with the other one, it would be nice to allow mapping the same warpd functionality to multiple keys (most importantly clicking a button and scrolling). 

At first I thought that this would be a job for keyd, but keeping state so that this mode is left properly in all possible scenarios would result in quite a mess (for example, besides knowing about oneshot-keys, warpd would need to track when hint-mode is entered so that it properly recognizes the `hint_exit` key). However, simple use cases as basic one-handed browsing or video watching exclusively in normal mode could of course be covered by reserving a special combination for having keyd enter such a "warpd one-handed mode" and then stay in normal mode only, but this somehow doesn't feel right.

Another simple but ugly solution would be to just introduce `buttons2`, `scrollup2` and `scrolldown2` options, but in case you see some usefulness in this functionality it should be implemented properly for all warpd keys. This would then of course require to change the config file syntax, for example by also accepting {kc1 kc2 ... } where currently just one key combination is allowed, keeping full backward compatibility.

**Comment by rvaiya** [comment]
I agree that expanding the config syntax to accommodate multiple keys is the correct solution. I will work on it when I have some time.

**Comment by rvaiya** [comment]
I had some spare time, so I decided to implement this. If an option is specified multiple times all supplied keys are considered valid. This applies to both single and multi key options. Additionally, supplying multiple keys to a single key option will have the same effect.
